lib/CMakeLists.txt

```diff
@@ -229,9 +268,6 @@ target_link_libraries(arango PUBLIC v8_interface)
 
 add_dependencies(arango v8_build)
 
-set(FROZEN_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/3rdParty/iresearch/external/frozen/include")
-set(FUNCTION2_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/3rdParty/iresearch/external/function2")
-
 target_include_directories(arango
```

Comment:
You forgot to remove this line

Reply:
don't get it. Why would I remove that line?